collections/src/main/scala/scalan/collections/Collections.scala

```diff
@@ -625,22 +640,15 @@ trait CollectionsDslStd extends impl.CollectionsStd with SeqsDslStd {
   }
 
   def pairColl_outerJoin[K, B, C, R](xs: PairColl[K, B], ys: PairColl[K, C], f: Rep[((B, C)) => R], f1: Rep[B => R], f2: Rep[C => R])
-                                    (implicit ordK: Ordering[K], selfType: Elem[Array[(K, R)]],
+                                    (implicit ordK: Ordering[K], nK: Numeric[K], selfType: Elem[Array[(K, R)]],
                                      eK: Elem[K], eR: Elem[R], eB: Elem[B], eC: Elem[C]): Rep[Array[(K, R)]] = {
```

Comment:
Yes in Std context it make sense to correlate it with the code generated by LmsCompilerScala